dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNLccConverterInterfaceIIDM.cpp

```diff
@@ -0,0 +1,141 @@
+//
+// Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```

Comment:
2020

Reply:
done, but I did that right after pushing :( Will be in next push